src/test/java/teammates/test/cases/action/InstructorEditInstructorFeedbackPageActionTest.java

```diff
@@ -139,6 +140,21 @@ protected InstructorEditInstructorFeedbackPageAction getAction(String... params)
     @Override
     @Test
     protected void testAccessControl() throws Exception {
-        //TODO: implement this
+        dataBundle = getTypicalDataBundle();
+        removeAndRestoreDataBundle(dataBundle);
```

Comment:
@whipermr5  In 

     @Override
    protected void prepareTestData() {
        super.prepareTestData();
        dataBundle = loadDataBundle("/InstructorEditStudentFeedbackPageTest.json");
        removeAndRestoreDataBundle(dataBundle);
    }

By `super.prepareTestData()` we can access the `TypicalDataBundle` by `super.databundle` but  in `testAccessControl`  we are calling the other base class methods like `verifyUnaccessibleWithoutLogin` which uses `dataBundle` and in this case it will use the `dataBundle` variable updated by overide method but we need to have the `TypicalDataBundle`
So that why its not working. 
I may not be right :sweat_smile: